css/disciplines-slider.styl

```diff
@@ -16,20 +16,26 @@ DISCIPLINE_COLOR_SOCIAL_SCIENCE = #7EA8CE
 
   .filmstrip--disciplines__discipline-card
     margin: 0.28em
+    border: 2px solid transparent
     text-align: center
     width: 5em
     font-size:25px
     height: 4em
     border-radius: 1em
-    border: 0
-    padding: .5em
+    padding: .2em .5em .5em
     color: #006DFF
     background-color: white
     min-width: 100px
-    &:hover,
+    &:active,
     &.filmstrip--disciplines__discipline-card--active
-      color: white
+      color: white !important
```

Comment:
Hmm that's a tricky one. It was there because the active font colour has to override the hover/focus font colour, and it has the same specificity so the hover/focus colour wins.

Managed to remove it by moving this block to the end, but I am not sure if that is bad practice in a stylus file as the `.filmstrip--disciplines__discipline-card` is now repeated...

```
.filmstrip--disciplines__discipline-card
 &:active,
 &.filmstrip--disciplines__discipline-card--active
   color: white
   outline: none
```